README.md

```diff
@@ -81,6 +81,27 @@ To avoid this problem, use the `--yes` flag to automatically confirm the prompts
 bevy build --yes web
 ```
 
+## Bevy scaffolding
+
+The `bevy new` command lets you easily scaffold a new Bevy project using a custom template or a minimal template provided by [Bevy](https://github.com/TheBevyFlock/bevy_new_minimal).
+Templates are just GitHub repositories and can be specified with the `-t` flag.
+
+### Usage
+
+If no template is omitted, the default minimal template will be chosen.
```

Comment:
```suggestion
If no template is omitted, the [default minimal template](https://github.com/TheBevyFlock/bevy_new_minimal) will be chosen.
```

I think it's good to link to the template here, in case the reader didn't notice it previously! This let's them check out the code before using the template :)